### PR TITLE
Preview and Transfer

### PR DESCRIPTION
**Pull request by rasika-purohit**
[skip ci]

### CODE REVIEW

File: docs/bot-builder/basic/transferbot.md [hunk 1]
```diff
@@ -1,35 +1,142 @@
...
+
+<hr>
+
+When someone is trying to transfer an empty bot i.e. a bot where Skill Set and Skill is not created, you will see a pop-up as shown below - 
+
+![image](https://user-images.githubusercontent.com/75118325/124418722-bf961f00-dd79-11eb-8514-ce0ef872e040.png)
+
+When there are no changes made to the bot since the last transfer, you will see a pop-up as shown below - 
+
+![image](https://user-images.githubusercontent.com/75118325/124419709-aee6a880-dd7b-11eb-833f-ab61c5d80043.png)
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:128]: For both scenarios, the image remains the same. The _refigerator_ one

File: docs/bot-builder/basic/transferbot.md [hunk 2]
```diff
@@ -1,35 +1,142 @@
...
+
+When a **Contributor** has saved new changes on the bot at the time when you initiated the transfer, you will get a pop-up to refresh the preview.
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:130]:
```suggestion
When another **Contributor** has saved new changes on the bot at the same time when you initiated the transfer, you will get a pop-up indicating the same. You can refresh the preview to see the new changes or go ahead with the transfer.
```

**Review by medhaanand — APPROVED** [review]

File: docs/bot-builder/basic/transferbot.md [hunk 3]
```diff
@@ -1,35 +1,140 @@
...
+
+Once the bot is built and has passed the functional testing, you can transfer the bot to production. 
+
+Click on the **Preview & Transfer** button. You will see a modal that generates a preview of changes as seen below - 
+
+![preview1](https://user-images.githubusercontent.com/75118325/124290857-91de8980-db71-11eb-96b2-dbee73b467f3.gif)
+
+The modal will display the changes that were made to the bot.
+
+![image](https://user-images.githubusercontent.com/75118325/124392874-5a104700-dd15-11eb-9f8a-435d0c690e77.png)
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:39]: Will have to change this image. It contains the grey icon

File: docs/bot-builder/basic/transferbot.md [hunk 4]
```diff
@@ -1,35 +1,140 @@
...
+
+> When you hover over the indicators in the **Change** section, you will be able to check the actions taken by the **Contributor** along with the timestamp.
+
+When a user had added a new element such as a new entity, new Skill Set, and so on, the indicator in the **Changes** section will be **_green_**.
+
+When some element had been deleted such as deactivating a node, removing, and so on, the indicator in the **Changes** section will be **_red_**.
+
+When there are small edits made to any element such as entity, the bot says, the indicator in the **Changes** section will be **_yellow_**.
+
+When there are any changes made to a Skill, such as changing the bot says, adding a new node, adding a new connection and so on, the indicator in the **Changes** section will be **_gray_**.
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:73]:
```suggestion
```

File: docs/bot-builder/basic/transferbot.md [hunk 5]
```diff
@@ -1,35 +1,140 @@
...
+
+![image](https://user-images.githubusercontent.com/75118325/124393354-b96f5680-dd17-11eb-96e1-a7390b497908.png)
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:75]: Screenshot will have to be changed

File: docs/bot-builder/basic/transferbot.md [hunk 6]
```diff
@@ -1,35 +1,140 @@
...
+
+### Changes to Entities
+
+You can check when a user has made any changes to the entity, for example, changing the entity settings or changing the entity reprompt message, and so on.
+
+When there are edits made to the already existing entity, the indicator will be **_yellow_**.
+
+![image](https://user-images.githubusercontent.com/75118325/124393523-8a0d1980-dd18-11eb-872e-929d741d3209.png)
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:83]: screenshot to be changed

File: docs/bot-builder/basic/transferbot.md [hunk 7]
```diff
@@ -1,35 +1,140 @@
...
+
+When a user is creating and adding a new entity, the indicator will be **_green_**. 
+
+![image](https://user-images.githubusercontent.com/75118325/124393566-b5900400-dd18-11eb-82d5-8417bbae11df.png)
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:87]: screenshot to be changed

File: docs/bot-builder/basic/transferbot.md [hunk 8]
```diff
@@ -1,35 +1,140 @@
...
+
+### Changes to Skill Sets
+
+The **_gray_** indicator present along Skill Set denotes there were changes made in the particular Skill Set.
+
+![image](https://user-images.githubusercontent.com/75118325/124417674-4c8ba900-dd77-11eb-8bc5-7951bb513533.png)
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:93]: Remove this entire section

File: docs/bot-builder/basic/transferbot.md [hunk 9]
```diff
@@ -1,35 +1,140 @@
...
+* Connections
+  * A new connection created between nodes is indicated as **_green_**.
+  * If a connection is deleted between nodes, it is indicated as **_red_**.
+  > 
+  > ![image](https://user-images.githubusercontent.com/75118325/124418127-65488e80-dd78-11eb-9445-14a13466c635.png)
+  > 
+
+When you hover over the indicator, you will be able to see the **Contributors** name. If there are multiple Contributors who made changes to a particular Skill Set, you can check their names and the changes they made to the particular Skill Set such as creating a new node, creating a new connection, etc.
+
+![image](https://user-images.githubusercontent.com/75118325/124418385-ffa8d200-dd78-11eb-93ca-0a346a590cae.png)
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:112]: screenshot to be changed.

File: docs/bot-builder/basic/transferbot.md [hunk 10]
```diff
@@ -1,35 +1,153 @@
...
+
+#### Indicators
+
+There are different indicators to denote different types of changes made on the bot. 
+
+> When you hover over the indicators in the **Change** section, you will be able to check the actions taken by the **Contributor** along with the timestamp.
+
+When a user had added a new element such as a new entity, new Skill Set, and so on, the indicator in the **Changes** section will be **_green_**.
+
+When some element had been deleted such as deactivating a node, removing, and so on, the indicator in the **Changes** section will be **_red_**.
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:67]:
```suggestion
When some element had been deleted such as deleting a connection, the indicator in the **Changes** section will be **_red_**.
```

File: docs/bot-builder/basic/transferbot.md [hunk 11]
```diff
@@ -1,35 +1,153 @@
...
+
+When there are small edits made to any element such as entity, the bot says, the indicator in the **Changes** section will be **_yellow_**.
```
Comment (RandiveSushant) [inline docs/bot-builder/basic/transferbot.md:69]:
```suggestion
When there are small edits made to any element such as the bot says of a node, the indicator in the **Changes** section for that particular element will be **_yellow_**.
```

**Review by medhaanand — APPROVED** [review]

**Pull request merged**